sdk/core/azure-core/src/main/java/com/azure/core/credential/SimpleTokenCache.java

```diff
@@ -185,4 +187,8 @@ private static LoggingEventBuilder buildTokenRefreshLog(LogLevel level, AccessTo
             .addKeyValue("retryAfterSeconds", REFRESH_DELAY_STRING)
             .addKeyValue("expired", tte.isNegative());
     }
+
+    void setRefreshDelay(Duration refreshDelayDuration) {
```

Comment:
Because this is only being used for testing purposes, but may be useful in the future, thoughts on making `refreshDelay` and `refreshDelayString` non-static final fields and add a package-private constructor that accepts `refreshDelay` and the current constructor passes a default of `Duration.ofSeconds(30)`.